.tool-versions

```diff
@@ -0,0 +1,3 @@
+mage 1.14.0
```

Comment:
Since we are adding this `.tool-versions` we might need to also update https://github.com/elastic/elastic-agent/blob/main/.github/workflows/bump-golang.yml

Alternativly, have the addition of the `.tool-versions` + workflow on a dedicated PR

Reply:
Do you mean we have to create a dedicated PR that with `.tool-version` update automation?

Reply:
There is an automation that bumps the golang version (mainly).

We need to ensure that it will also include this file when this happens.

Reply:
Created a follow-up [issue](https://github.com/elastic/ingest-dev/issues/4302)